src/main/java/org/reactome/qa/doisuggester/Main.java

```diff
@@ -88,6 +89,12 @@ public static void main(String[] args) throws FileNotFoundException, IOException
 			printOutput(pathwaysMap);
 
 		}
+		catch (PropertyHasNoValueException | PropertyNotPresentException e)
+		{
+			System.err.println("Your properties file appears to be misconfigured: " + e.getMessage());
+			e.printStackTrace();
```

Comment:
No problems here, but I just wanted to double-check you mentioned you'll be adding a logger in future changes?

Reply:
Yes. That is out of scope for this PR.

Reply:
Yeah, that's okay.  No need to change it now - I just wanted to double-check it will be done.

Reply:
Yes, I'm just trying to keep the PRs smaller, with more limited scopes. This one is ONLY for the MandatoryProperties usage. Logging improvements are _out of scope_.